internal/driver/glfw/driver_desktop.go

```diff
@@ -13,7 +13,10 @@ import (
 	"fyne.io/fyne/v2/theme"
 )
 
-var setup sync.Once
+var (
+	iconSet fyne.Resource
```

Comment:
The name `iconSet` is misleading. My first thought was “a set of icons”. Even after realising that this should be a set icon, I did not know what to make of it until reading the whole code.
I suggest `systrayIcon` instead.